tests/Core/Generators/Fixtures/MarkdownDouble.php

```diff
@@ -35,4 +35,34 @@ public function getRealFooter()
     {
         return parent::getFormattedFooter();
     }
+
+    /**
+     * [VISIBILITY WIDENING ONLY] Print the header of the HTML page.
```

Comment:
Why are we making deprecated methods _more_ visible?

Reply:
@fredden These are test doubles only. It allows for verifying that the deprecation notices are shown via the automated tests.
Of course, we could use Reflection for that, but as these tests double were in place anyway, just overloading the methods with `public` visibility made creating the tests more straight forward.

Both those tests + the method overloads in the doubles will be removed in 4.0 (alongside the removal of the deprecated methods).